### PR TITLE
vim-patch:0e6adf8a29d5

### PR DESCRIPTION
**Pull request by clason**
Update runtime files
https://github.com/vim/vim/commit/0e6adf8a29d5c2c96c42cc7157f71bf22c2ad471

skip runtime/optwin.vim (included in #16618)
skip runtime/map.vim (needs 8.2.3619)

### CODE REVIEW

**Comment by clason** [comment]
@seandewar double whammy for you: `termdebug.vim` changes making use of method syntax :)

File: runtime/doc/map.txt [hunk 1]
```diff
@@ -847,28 +847,33 @@ Here is an example that counts the number of spaces with <F4>: >
 	" doubling <F4> works on a line
 	nnoremap <expr> <F4><F4> CountSpaces() .. '_'
 
-	function CountSpaces(type = '') abort
+	function CountSpaces(virtualedit = '', irregular_block = v:false, type = '') abort
 	  if a:type == ''
-	    set opfunc=CountSpaces
+	    let &operatorfunc = function('CountSpaces', [&virtualedit, v:false])
```
[the remark this answers is not in the document]
Comment (clason) [reply runtime/doc/map.txt:852]: ah, I was afraid of that... better to revert, then?

[the remark this answers is not in the document]
Comment (seandewar) [reply runtime/doc/map.txt:852]: I think so :+1:

**Comment by seandewar** [comment]
Missing https://github.com/vim/vim/commit/0e6adf8a29d5c2c96c42cc7157f71bf22c2ad471#diff-f35d708d4c55517f9599b4a8f7a37a2089f32f421791055318bf1a3529b7abd9R176 but otherwise LGTM!

**Review by seandewar — APPROVED** [review]

**Comment by clason** [comment]
(macOS CI was a network timeout; Windows is the usual cereal flakes)

**Pull request merged**